contents/docs/product-analytics/troubleshooting.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Troubleshooting and FAQs
+---
+
+Having trouble with events? There are a few common reasons that you may not see events appear in your project.
+
+### 1. The library is being used incorrectly. (this is usually the issue)
+
```

Comment:
It would be good to have some sort of introduction here describing what we'll do to debug:

e.g., "It's possible that you have PostHog is not initializing correctly due to misconfiguration (or whatever the reason is?). To debug if this is the case, we'll do xyz"